PGD: clarify bdr.join_node_group()'s synchronize_structure parameter (#5164)
EnterpriseDB/docs: product_docs/docs/pgd/5/reference/nodes-management-interfaces.mdx

-   `wait_for_completion` &mdash; Wait for the join process to complete before
    returning. Defaults to `true`.
-   `synchronize_structure` &mdash; Set the kind of structure (schema) synchronization

>     to do during the join. Valid options are `all` (the default), which synchronizes

Reviewer: ```suggestion
    to do during the join. The default setting is `all` (the default), which synchronizes
```

>     the complete database structure, and `none`, which doesn't synchronize any

Reviewer: ```suggestion
    the complete database structure. The other available setting is `none` which doesn't synchronize any
```

>     structure. However, it still synchronizes data (except for witness nodes).

Reviewer: ```suggestion
    structure. However, it still synchronizes data (except for witness nodes, which by design do not synchronize data).
```
